ddtrace/internal/debug.py

```diff
@@ -137,3 +137,82 @@ def collect(tracer):
+        info.get("agent_error") or "None",
+        info.get("analytics_enabled"),
+        info.get("log_injection_enabled"),
+        info.get("health_metrics_enabled"),
+        info.get("priority_sampling_enabled"),
+        info.get("partial_flush_enabled"),
+        info.get("partial_flush_min_spans") or "Not set",
+        bcolors.OKGREEN,
+        bcolors.BOLD,
+        bcolors.ENDC,
+        info.get("service") or "None",
+        info.get("env") or "None",
+        info.get("dd_version") or "None",
+        info.get("global_tags") or "None",
+        info.get("tracer_tags") or "None",
+    )
+
+    summary = "{0}{1}Summary{2}".format(bcolors.OKCYAN, bcolors.BOLD, bcolors.ENDC)
+
+    if info.get("agent_error"):
+        summary += "\n\n{0}ERROR: It looks like you have an agent error: '{1}'\nThe most common error is a connection error. If you're experiencing a connection error, please make sure you've followed the setup for your particular environment so that the tracer and Datadog agent are configured properly to connect, and that the Datadog agent is running: https://docs.datadoghq.com/tracing/setup_overview/setup/python/?tab=containers#configure-the-datadog-agent-for-apm\nIf your issue is not a connection error then please reach out to support for further assistance: https://docs.datadoghq.com/help/{2}".format(
+            bcolors.FAIL, info.get("agent_error"), bcolors.ENDC
+        )
+
+    if not info.get("service"):
+        summary += "\n\n{0}WARNING SERVICE NOT SET: It looks like you haven't set a service tag for this service. We'd recommend setting one with the environment variable DD_SERVICE as it's used for the scoping of application specific data across metrics, traces, and logs. For more information see https://docs.datadoghq.com/getting_started/tagging/unified_service_tagging/?tab=kubernetes#overview{1}".format(
+            bcolors.WARNING, bcolors.ENDC
+        )
+
+    if not info.get("env"):
+        summary += "\n\n{0}WARNING ENV NOT SET: It looks like you haven't set an env tag for this service. We'd recommend setting one with the environment variable DD_ENV as it's used for the scoping of the application's data to a specific environment, e.g. env:prod vs env:dev. For more information see https://docs.datadoghq.com/getting_started/tagging/unified_service_tagging/?tab=kubernetes#overview{1}".format(
+            bcolors.WARNING, bcolors.ENDC
+        )
+
+    if not info.get("dd_version"):
+        summary += "\n\n{0}WARNING VERSION NOT SET: It looks like you haven't set a version tag for this service. We'd recommend setting one with the environment variable DD_VERSION as it's used to scope an application's data to a specific version of that application, e.g. comparing version:0.34.0 vs version:0.35.0. For more information see https://docs.datadoghq.com/getting_started/tagging/unified_service_tagging/?tab=kubernetes#overview{1}".format(
```

Comment:
I wonder if it would be better to point to our docs rather than corp docs since we can be sure the url never changes?

Reply:
Do we have docs talking about unified service tagging or why a customer would want to set things like version? I doubt the unified service tagging docs are going anywhere soon.

Reply:
We probably need to add it to our API docs

Like, we could add more entries here to link to?

https://ddtrace.readthedocs.io/en/stable/troubleshooting.html

Reply:
I could pull from different places in our docs/ take what I removed from the summaries in the command and add that to API docs. Do you think the troubleshooting page would be the best place for that though? It might work if we added the --status command to the troubleshooting page and then we could take about why you'd want to set those things if you go those warnings.